src/lib/homestore/heap_chunk_selector.cpp

```diff
@@ -117,4 +117,16 @@ void HeapChunkSelector::build_per_dev_chunk_heap(const std::unordered_set< chunk
         if (excludingChunks.find(p.first) == excludingChunks.end()) { add_chunk_internal(p.first); }
     };
 }
-} // namespace homeobject
\ No newline at end of file
+
+homestore::blk_alloc_hints HeapChunkSelector::get_blk_alloc_hints_on_same_pdev(chunk_num_t chunk_id) const {
```

Comment:
chunk_id can be const.

Reply:
I believe we decided not to use `const` for parameter by value as it very verbose especially once you have more parameters and provide very little value if any.

Reply:
yes; i personally can not think of every a time where `const T` would have prevented some bug. (`const T&` obviously i have).